src/humancellatlas/data/metadata/api.py

```diff
@@ -62,6 +63,8 @@ def __init__(self, json: JSON) -> None:
         self.json = json
         provenance = json.get('hca_ingest') or json['provenance']
         self.document_id = UUID4(provenance['document_id'])
+        submitter_id = provenance.get('submitter_id')
+        self.submitter_id = None if submitter_id is None else UUID4(submitter_id)
```

Comment:
The schema types this as a general string, without a pattern. I don't think we should assume that it is a UUID.